homeassistant/components/denonavr/media_player.py

```diff
@@ -188,49 +238,55 @@ def device_info(self):
     @property
     def name(self):
         """Return the name of the device."""
-        return self._name
+        return self._receiver.name
 
     @property
     def state(self):
         """Return the state of the device."""
-        return self._state
+        if self._available is False:
+            return STATE_UNAVAILABLE
+        if self._receiver.state is None:
+            return STATE_UNKNOWN
```

Comment:
```suggestion
            return None
```

We use `None` for unknown state

Reply:
ok, done.
Btw. what's the purpose of the constant then?